config/configmap/inferenceservice.yaml

```diff
@@ -31,7 +31,7 @@ data:
             "supportedFrameworks": [
               "sklearn"
             ],
-            "multiModelServer": false
+            "multiModelServer": true
```

Comment:
Should sklearn v2 `multiModelServer` flag be true as well?

Reply:
There is no doc and e2e test currently for v2 MLServer mms, @adriangonz What's the current mms status for MLServer?

Reply:
There should be e2e tests for both v1 and v2 in this file (they got introduced in #1322): 

https://github.com/kubeflow/kfserving/blob/master/test/e2e/predictor/test_multi_model_serving.py

Regarding the docs, that's a good question. I remember adding an example for the individual models, but I don't think we got around adding an example with MMS. What would be the best place to add these? Should we open this as a separate issue?

Reply:
@adriangonz Nice ! Can you help add the examples [here](https://github.com/kubeflow/kfserving/tree/master/docs/samples/multimodelserving/sklearn), move the current one to v1 folder and added yours in v2?